### ISSUE TITLE
CPack/Installer should copy `vegastrike.config` into `$HOME/.vegastrike`

### ISSUE THREAD

**Issue by stephengtuggy** [vegastrike/Assets-Production#49]
**Describe the bug**
The installer doesn't currently copy the vegastrike.config file into ~/.vegastrike, only into /usr/share/vegastrike. So users have to copy it manually, or use another method to get the file into the proper directory.

**To Reproduce**

1. Install the Vega Strike engine, v0.7.0 beta 10, followed by the Assets beta. 
2. Try running vegastrike or vsettings.

**Expected behavior**
There should already be a default `vegastrike.config` file in the `$HOME/.vegastrike` directory to start with. Both vsettings and vegastrike binaries should "just work" with that file, albeit with some limitations in terms of game resolution, etc.

**Please complete the following information):**
 - OS: Reported on Ubuntu 20.04 LTS.
 - Software Versions: Reported with v0.7.0-beta10

**Additional context**
Originally reported as https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/417#issuecomment-774571108
Then filed as https://github.com/vegastrike/Vega-Strike-Engine-Source/issues/420
Moving the issue to the correct (I think) repository.

**Comment by BenjamenMeyer** [comment]
I'm not sure this is something the installer can do; rather the default probably should be at (for example) `/usr/share/vegastrike/config`, and then copied over to the user's profile (~/.vegastrike) only when saved.

Reason being: the installer doesn't know anything about the user, especially on Linux; and we should try to be consistent across platforms.

**Comment by stephengtuggy** [comment]
Hmm. I see your point.

I guess this raises another question: Should Vega Strike / vsUTCS be a system-wide installation? A per-user installation? Should they support both configurations?

**Comment by BenjamenMeyer** [comment]
@stephengtuggy  yes it should be system wide by design; though the user's installation system (at least on Windows) may offer a user-only type installation. I don't think Linux Package Managers typically provide for a user-specific install. Not sure about Mac. AFAIK only Windows does, but even then it might fall to the installer to decide if that's an option (likely) in which case we should support it if we could.

**Comment by stephengtuggy** [comment]
Filed a follow-up issue, #50

**Comment by BenjamenMeyer** [comment]
We need to get 0.7.x out the door; deferring this as it'll be more research oriented to determine an answer

**Comment by BenjamenMeyer** [comment]
as we're getting ready to release 0.7.0 and we want a fast turn around on 0.8.x i'm deferring this to 0.9.x
